actions/setup-sigstore-env/run-containers.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+#
+# Copyright 2025 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+echo "setting up kind"
+go install sigs.k8s.io/kind@v0.20.0
+kind create cluster
```

Comment:
Why do we need `kind`?

Reply:
I see it's actaully cosign-specific. removing.

 - https://github.com/sigstore/cosign/commit/8b0bcecfeb824e74fbad6607c719e77d628c3aab#diff-9e762d0df1502ee31a1de1a3bc67be6c2707ec3e6117be3147907a60ef5fb854R231